e2e-tests/playwright/support/test_fixture.ts

```diff
@@ -65,9 +67,14 @@ class PlaywrightExtended {
     // ./visual
     readonly matchSnapshot;
 
+    // ./mock_browser_api
+    readonly stubNotification;
+    readonly waitForNotification;
+
     constructor(browser: Browser) {
         // ./browser_context
-        this.testBrowser = new TestBrowser(browser);
+        this.TestBrowser = _TestBrowser;
+        this.testBrowser = new _TestBrowser(browser);
```

Comment:
Oh right, good catch! I initially thought of exposing `TestBrowser` in `pw` so it can be used as needed. In this PR, the use case is for multiple isolated browser sessions but the use of instantiated `testBrowser` works perfectly fine. I reverted the change.

Reply:
Thanks this is great improvement